examples/fewshot_protonet/configs/template.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a comprehensive config file for protonet.
+# Users can modify all changeable hyperparameters to train their own model.
+DATASET:
+  ROOT: 'data/omniglot'
```

Comment:
Please check and fix all. We use double quotes for strings. See other examples of PyKale.